h2o-r/h2o-package/R/communication.R

```diff
@@ -73,6 +212,15 @@
 
   url = .h2o.calcBaseURL(conn, h2oRestApiVersion = h2oRestApiVersion, urlSuffix = urlSuffix)
 
+  if (!getOption("prefer_RCurl", FALSE) && requireNamespace("curl"))
```

Comment:
Do we have the curl package available in test docker images?

Reply:
for tests it is handy to have an option to force certain behavior (and fail when the requirements are not satisfied instead of falling back to rcurl)

Reply:
the main concern is the hadoop images where authentication is tested

Reply:
> Do we have the curl package available in test docker images?
We should have it there: Dockerfile-build-hadoop depends on Dockerfile-build-base which depends on Dockerfile-r which installs using `install_R_version` the `devtools` which depends on the `curl` packages.

I didn't run hadoop tests on this PR to check but I checked that ordinary R tests are run in docker with the `curl` package.

Reply:
Ok, np, I checked one of the images now and it seems to be there - I think we are all good.